script-runner/AutomatedTestRunner.hs

```diff
@@ -0,0 +1,400 @@
+{-# LANGUAGE ApplicativeDo   #-}
+{-# LANGUAGE ConstraintKinds #-}
+{-# LANGUAGE DeriveGeneric   #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving   #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE NamedFieldPuns #-}
```

Comment:
We mentioned that in Slack. If its not needed any longer, its probably best to remove it (same with other extensions)